task12/index.js

```diff
@@ -0,0 +1,56 @@
+const parent = document.getElementById("js-parent");
+    
+const appendButtonForParent = () => {
+    const button = document.createElement('input');
+    button.id = "button";
+    button.type = "button";
+    button.value = "click";
+    parent.appendChild(button);
+};
+appendButtonForParent();
+
+const removeButton = () => {
+    document.getElementById("button").remove();
+};
+
+const addEvent = document.getElementById("button"); 
+addEvent.addEventListener('click', () => {
+    removeButton();
+    init();
+});
+
+const getData = async () => {
+    try {
+        const request = await fetch("https://jsondata.okiba.me/v1/json/3jhPW210812052923");
```

Comment:
NITS：

``` const request ...```

The variable name 'request' is I thought it would be better to name it something like `res` or `response`,
Because it is assumed that the content of the response or rather the request will come.

Reply:
Thank you for pointing out the variables.
I learned a lot!